basxconnect/mailer_integration/synchronize.py

```diff
@@ -14,9 +16,21 @@
 def synchronize(mailer: AbstractMailer) -> SynchronizationResult:
     synchronize_interests(mailer)
 
-    mailer_persons = mailer.get_persons()
-    datasource_tag = _get_or_create_tag(mailer.tag())
+    person_count = mailer.get_person_count()
     sync_result = SynchronizationResult.objects.create()
+    sync_result.total_synchronized_persons = person_count
+    batch_size = 1000
+    for batch_number in range(int(math.ceil(person_count / batch_size))):
```

Comment:
FYI "C-style" math instead of the math module would also work here:

    (person_count // batch_size) + 1

Reply:
Right, of course

We could adjust for it with ```((i-1) // batch_size) + 1``` but then the "avoiding floating point" starts to loose again "readability" ;)

Although I am writing now (slow) web applications with Python for many years I somehow still feel uncomfortable when I see "avoidable" use of floats :D

Reply:
I agree that it is better to avoid floating point. It also gets rid of an import statement. I updated it now.